### ISSUE TITLE
[2.1] FEE Error on widget

### ISSUE THREAD

**Issue by zonky2** [MetaModels/attribute_tabletext#24]
Error at FE

```
Fatal error: Class name must be a valid object or a string in C:\...\contao-3.5.x-fee\composer\vendor\contao-community-alliance\dc-general-contao-frontend\src\View\DefaultWidgetBuilder.php on line 115
```
https://github.com/contao-community-alliance/dc-general-contao-frontend/blob/654c6017b59dd3257897aa69c5187f7ac0b74213/src/View/DefaultWidgetBuilder.php#L115

**Comment by zonky2** [comment]
at MM 2.1

```
ContaoCommunityAlliance\DcGeneral\Exception\DcGeneralRuntimeException:
Widget was not build for property mm_employees::soft_skills.

  at vendor/contao-community-alliance/dc-general-contao-frontend/src/View/WidgetManager.php:103
  at ContaoCommunityAlliance\DcGeneral\ContaoFrontend\View\WidgetManager->getWidget('soft_skills', null)
     (vendor/contao-community-alliance/dc-general-contao-frontend/src/View/WidgetManager.php:127)
  at ContaoCommunityAlliance\DcGeneral\ContaoFrontend\View\WidgetManager->renderWidget('soft_skills', false, null)
     (vendor/contao-community-alliance/dc-general-contao-frontend/src/View/EditMask.php:389)
  at ContaoCommunityAlliance\DcGeneral\ContaoFrontend\View\EditMask->buildFieldSet(object(WidgetManager), object(Palette), null)
     (vendor/contao-community-alliance/dc-general-contao-frontend/src/View/EditMask.php:180)
  at ContaoCommunityAlliance\DcGeneral\ContaoFrontend\View\EditMask->execute()
     (vendor/contao-community-alliance/dc-general-contao-frontend/src/View/ActionHandler/EditHandler.php:120)
  at ContaoCommunityAlliance\DcGeneral\ContaoFrontend\View\ActionHandler\EditHandler->process(object(DefaultEnvironment))
     (vendor/contao-community-alliance/dc-general-contao-frontend/src/View/ActionHandler/EditHandler.php:80)
  at ContaoCommunityAlliance\DcGeneral\ContaoFrontend\View\ActionHandler\EditHandler->handleEvent(object(ActionEvent), 'dc-general.action', object(TraceableEventDispatcher))
     (vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php:111)
  at Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke(object(ActionEvent), 'dc-general.action', object(ContainerAwareEventDispatcher))
     (vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:212)
  at Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(array(object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener), object(WrappedListener)), 'dc-general.action', object(ActionEvent))
     (vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:44)
  at Symfony\Component\EventDispatcher\EventDispatcher->dispatch('dc-general.action', object(ActionEvent))
     (vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php:143)
  at Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch('dc-general.action', object(ActionEvent))
     (vendor/contao-community-alliance/dc-general-contao-frontend/src/FrontendEditor.php:86)
  at ContaoCommunityAlliance\DcGeneral\ContaoFrontend\FrontendEditor->editFor('mm_employees', 'create')
     (vendor/metamodels/contao-frontend-editing/src/FrontendEditHybrid.php:92)
  at MetaModels\ContaoFrontendEditingBundle\FrontendEditHybrid->compile()
     (vendor/contao/core-bundle/src/Resources/contao/classes/Hybrid.php:236)
  at Contao\Hybrid->generate()
     (vendor/metamodels/core/src/FrontendIntegration/MetaModelHybrid.php:213)
  at MetaModels\FrontendIntegration\MetaModelHybrid->generate()
     (vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:480)
  at Contao\Controller::getContentElement(object(ContentModel), 'main')
     (vendor/contao/core-bundle/src/Resources/contao/modules/ModuleArticle.php:183)
  at Contao\ModuleArticle->compile()
     (vendor/contao/core-bundle/src/Resources/contao/modules/Module.php:220)
  at Contao\Module->generate()
     (vendor/contao/core-bundle/src/Resources/contao/modules/ModuleArticle.php:65)
  at Contao\ModuleArticle->generate(false)
     (vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:420)
  at Contao\Controller::getArticle(object(ArticleModel), false, false, 'main')
     (vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:279)
  at Contao\Controller::getFrontendModule('0', 'main')
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:174)
  at Contao\PageRegular->prepare(object(PageModel))
     (vendor/contao/core-bundle/src/Resources/contao/pages/PageRegular.php:47)
  at Contao\PageRegular->getResponse(object(PageModel), true)
     (vendor/contao/core-bundle/src/Resources/contao/controllers/FrontendIndex.php:306)
  at Contao\FrontendIndex->renderPage(object(Collection))
     (vendor/contao/core-bundle/src/Resources/contao/controllers/FrontendIndex.php:75)
  at Contao\FrontendIndex->run()
     (vendor/contao/core-bundle/src/Controller/FrontendController.php:42)
  at Contao\CoreBundle\Controller\FrontendController->indexAction()
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:151)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php:68)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:200)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web/app_dev.php:64)
```

**Comment by zonky2** [comment]
Fixed in MM 2.2
